AcadeMeData/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.9 on 2021-11-22 22:06
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
```

Comment:
This one is created auto by make migration,
If you clean your db.sqlite file, this part will not be created

Reply:
Do not forget to remove lines 10-12,
see my other comment in the other test file related to dependencies

Reply:
changes you requested return errors - please chat with me on slck

Reply:
You are correct, my mistake